### PR TITLE
Db based on diagram PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

In this branch me and @Nezerwa,

- [ ] Created a file named schema_based_on_diagram.sql where we implemented the database from the diagram
- [ ] Joined tables from many-to-many relationships might not appear in the diagram
- [ ] Added the FK indexes

Looking forward to your response.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  treatment_id INT NOT NULL,
+  CONSTRAINT fk_medical_histories_treatments_medical_histories
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+  CONSTRAINT fk_medical_histories_treatments_treatments
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+CREATE INDEX idx_medical_histories_treatments_medical_history_id
+    ON medical_histories_treatments (medical_history_id);
+CREATE INDEX idx_medical_histories_treatments_treatment_id
+    ON medical_histories_treatments (treatment_id);
```
Comment (mckent05) [inline schema_based_on_diagram.sql]: - According to the project requirements, You are to create an Index on ALL foreign keys. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @mertkantaroglu, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating the database based on the diagram 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (mertkantaroglu) [reply schema_based_on_diagram.sql]: Thanks @mckent05 👍

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Please avoid pushing any changes to the repo after submitting for review, this might cause the review to be incorrect.
## Highlights
- great job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mertkantaroglu** [comment]
Thanks @mikethreels 👍

**Pull request merged**